src/domain/primitives/tx_denomination_currency.types.d.ts

```diff
@@ -0,0 +1 @@
+type TxDenominationCurrency = "USD" | "BTC"
```

Comment:
this cant be a type, fiat currencies cant be hardcoded (this was a "intiial" implementation from nicolas but never was used)